domains/strychnine.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "strychine",
```

your github username
```suggestion
    "username": "pratik1120",
```